bundles/binding/org.openhab.binding.samsungac/src/main/java/org/openhab/binding/samsungac/internal/AirConditioner.java

```diff
@@ -230,7 +245,7 @@ private void writeLine(String line) throws Exception {
             writer.newLine();
             writer.flush();
         } catch (Exception e) {
-            logger.info("Could not write line. Disconnecting..., exception was: " + e);
+            logger.debug("Could not write line. Disconnecting..., exception was: " + e);
```

Comment:
Please use either parameterized logging, or the method signature that takes a `Throwable` as second argument.